checks/categories.py

```diff
@@ -2182,8 +2183,8 @@ def __init__(self):
             label="detail mail auth spf label",
             explanation="detail mail auth spf exp",
             tech_string="detail mail auth spf tech table",
-            worst_status=scoring.MAIL_AUTH_SPF_WORST_STATUS,
-            full_score=scoring.MAIL_AUTH_SPF_PASS,
+            worst_status=scoring.MAIL_AUTH_TLSRPT_WORST_STATUS,
```

Comment:
Seems like this should stay at SPF, and *_TLSRPT_* should be used in MailAuthTlsRptExists